lib/galaxy/managers/histories.py

```diff
@@ -215,7 +215,8 @@ def __init__( self, app ):
             'contents_url',
             # 'hdas',
             'empty',
-            'size', 'nice_size',
+            'size',
+            # 'nice_size',
```

Comment:
So I checked blend4j and bioblend objects since they each have places they expect specific attributes on objects and neither reference ``nice_size``. I am :+1: on this.